src/site/stages/build/process-cms-exports/tests/transformed-entities/node-outreach_asset.json

```diff
@@ -0,0 +1,23 @@
+{
+  "changed": 1565131126,
+  "entityId": "556",
+  "fieldBenefits": null,
+  "fieldDescription": "Rob transferred his Post-9/11 GI Bill benefits to his college-bound son.",
+  "fieldFormat": "video",
+  "fieldMedia": {
+    "entity": {
+      "entityBundle": "video",
+      "fieldMediaVideoEmbedField": "https://www.youtube.com/watch?v=9rCwFtxMaE0",
+      "thumbnail": {
+        "derivative": {
+          "url": "https://dev.cms.va.gov/sites/default/files/styles/large/public/video_thumbnails/9rCwFtxMaE0.jpg?itok=p4JEIa-a"
+        }
+      }
+    }
+  },
+  "fieldOffice": {
+    "targetId": 735
```

Comment:
Getting this might be tricky since the exported entity doesn't seem to have a `targetId` or anything corresponding to 735.  It seems like the only place this is used is here:

https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/layouts/publication_listing.drupal.liquid#L111